docs/dotprompt.md

```diff
@@ -164,22 +164,22 @@ output:
     description: string
 ---
 
-Generate a tabletop RPG character that would be found in {{location}}.
+Generate a restaurant menu that would be found in {{location}}.
```

Comment:
This sample is showcasing the ability to coerce the LLM to provide complex JSON output. As such, the schema above (L159-164) specifies an output object with 3 fields `name`, `hitPoints`, `description.`.

If we want to go with the menu theme, we should update the schema with a few different properties, and perhaps tweak the prompt.

We could do something like:

```
---
model: vertexai/gemini-1.0-pro
input:
  schema:
    theme: string
output:
  format: json
  schema:
    name: string
    price: integer
    ingredients(array): string
---

![image](https://github.com/firebase/genkit/assets/2858322/579f29e9-427d-4419-bbce-b2895dba74bf)


Generate a menu item that could be found at a {{theme}} restaurant.
```

Reply:
Prompt in action. 

![image](https://github.com/firebase/genkit/assets/2858322/d9df4319-2188-45df-97df-25bf8df09849)